vdyp-fip/src/test/java/ca/bc/gov/nrs/vdyp/fip/FipStartTest.java

```diff
@@ -1821,6 +1825,8 @@ static BiFunction<Integer, Integer, Optional<Coefficients>> closeUtilMap(int gro
 					return Optional.of(new Coefficients(new float[] { -0.796000004f, 0.141299993f, 0.0033499999f }, 1));
 				case 4:
 					return Optional.of(new Coefficients(new float[] { 2.35400009f, 0.00419999985f, 0.0247699991f }, 1));
+				default:
```

Comment:
I think I made these throw in the default case to guard against an invalid test state.  Did you confirm that the test is still valid?

Reply:
The code I changed looks to have disappeared. But I did confirm that the test ran before committing...